main/autobot/project_tracker.js

```diff
@@ -0,0 +1,37 @@
+var Handler = require('../autobot/handler');
+var JiraApi = require('jira').JiraApi;
+
+var JiraTracker = function (config) {
```

Comment:
I feel like this should live on its own, along with each other potential tracker. This way we aren't importing a crap ton of configurations for each tracker for one file.